src/wallet/walletdb.cpp

```diff
@@ -861,11 +863,15 @@ DBErrors WalletBatch::FindWalletTx(std::vector<uint256>& vTxHash, std::list<CWal
             // Read next record
             CDataStream ssKey(SER_DISK, CLIENT_VERSION);
             CDataStream ssValue(SER_DISK, CLIENT_VERSION);
-            int ret = m_batch.ReadAtCursor(pcursor, ssKey, ssValue);
-            if (ret == DB_NOTFOUND)
+            bool complete;
+            bool ret = m_batch.ReadAtCursor(ssKey, ssValue, complete);
+            if (complete) {
+                m_batch.CloseCursor();
                 break;
-            else if (ret != 0)
+            }
```

Comment:
Nit: coding style `} else if (!ret) {`

Reply:
Fixed.

Reply:
15tNMnu84FEsYbsavhHKmQTsNBCZBPg5EY
https://freebitco.in/#
https://btcdirect.eu/fr-be/cours-bitcoin
https://www.blockchain.com/btc/payment_request?address=12dRugNcdxK39288NjcDV4GX7rMsKCGn6B